### DISCUSSION TITLE
How to write $A \cup B$ or $A \cap B$ Latex in Typst

### DISCUSSION THREAD

**Discussion by witopea** [typst/typst#3282] [General]
How to write $A \cup B$ or $A \cap B$ Latex in Typst
Thanks

**Comment by PgBiel** [comment]
You can use `$A union B$` and `$A sect B$`

For more symbols available by default in Typst, see https://typst.app/docs/reference/symbols/sym/

If any symbol isn't in that page, you can also just paste it directly as a unicode character.

**Reply by witopea** [reply]
Thank you. Very much.:-)
